tutorials/full_ecModel/protocol.m

```diff
@@ -318,7 +395,57 @@
 % % Export to Excel file (will not contain potential model.ec content)
 % exportToExcelFormat(ecModel,'filename.xlsx');
 
-% STEP 22 Selecting objective functions
+% STEP 24 Simulate Crabtree effect with protein pool
+
+% (Re)load the ecModel without proteomics integration
+ecModel = loadEcModel('ecYeastGEM.yml');
+% Or reset the enzyme concentrations:
```

Comment:
Line 402-404: resetting the enzyme concentrations cannot completely revert back the model as some exchange reactions are constrained by Step 21?
I would suggest deleting lines 402-404.

Reply:
Good point!